src/StatusQ/Layout/StatusAppTwoPanelLayout.qml

```diff
@@ -5,7 +5,7 @@ import QtQuick.Controls 2.13
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 
-Item {
+FocusScope {
```

Comment:
Can you elaborate how this change is needed for the TextArea?

Reply:
It's needed in order to put initial focus on the first StatusTextArea; otherwise it will stay somewhere at the toplevel (I think the navbar has it).